analyses/cellranger-analysis/waiter.sh

```diff
@@ -7,12 +7,14 @@
 #BSUB -R "rusage[mem=2500]"
 #BSUB -o waiter.out
 #BSUB -e waiter.err
-#BSUB -cwd "./sc-rna-seq-snap/analyses/cellranger-analysis"
-
+#BSUB -cwd "/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/pipeline-testing/sc-rna-seq-snap/analyses/cellranger-analysis"
```

Comment:
Just an extra reminder to remove these specific paths when merging

Reply:
This has been addressed here 6c271f5863d698cbb52bd628dcf3842cf2313baf.